entrypoint.sh

```diff
@@ -35,9 +30,29 @@ if ! APTIBLE_OUTPUT_FORMAT=json aptible apps | jq -e ".[] | select(.handle == \"
   exit 1
 fi
 
-aptible deploy --environment "$INPUT_ENVIRONMENT" \
-               --app "$INPUT_APP" \
-               --docker-image "$INPUT_DOCKER_IMG" \
-               --private-registry-username "$INPUT_PRIVATE_REGISTRY_USERNAME" \
-               --private-registry-password "$INPUT_PRIVATE_REGISTRY_PASSWORD" \
-               ${INPUT_CONFIG_VARIABLES}
+if [ "$INPUT_TYPE" == "git" ]; then
+  export ACCESS_TOKEN=$(cat "$HOME/.aptible/tokens.json" | jq '.["https://auth.aptible.com"]' -r)
+  REMOTE_URL="root@$INPUT_GIT_REMOTE:$INPUT_ENVIRONMENT/$INPUT_APP.git"
+  git remote add aptible ${REMOTE_URL}
+  REMOTE_BRANCH="deploy-$(date "+%s")"
+  GIT_SSH_COMMAND="ssh -o SendEnv=ACCESS_TOKEN -o PubkeyAuthentication=no -o StrictHostKeyChecking=no -p 43022" git push aptible "$GITHUB_BRANCH:$REMOTE_BRANCH"
```

Comment:
> StrictHostKeyChecking=no
We should not skip hostkey checking from a location outside of our control, eg GHA runners. (this will necessitate persisting it between deployments of Primetime with a PersistentDisk)

Reply:
Updated.  Will have to re-test this to make sure it all works still